sim/warlock/items.go

```diff
@@ -220,3 +221,48 @@ var ItemSetGladiatorsFelshroud = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+// T13
+var ItemSetVestmentsOfTheFacelessShroud = core.NewItemSet(core.ItemSet{
+	Name: "Vestments of the Faceless Shroud",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			warlock := agent.(WarlockAgent).GetWarlock()
+
+			warlock.AddStaticMod(core.SpellModConfig{
+				Kind:      core.SpellMod_Cooldown_Flat,
+				TimeValue: -time.Minute * 4,
+				ClassMask: WarlockSpellSummonDoomguard | WarlockSpellSummonInfernal,
+			})
+
+			warlock.Tier132PSummonDuration = core.TernaryInt32(warlock.Spec == proto.Spec_SpecDemonologyWarlock, 20, 30)
```

Comment:
Nit but it's better calling the boni 2pT13 so that there's seperation between the digits